common/mb_motor.c

```diff
@@ -122,6 +161,7 @@ int mb_motor_set_all(double duty){
 * returns the measured current in A
 *******************************************************************************/
 double mb_motor_read_current(int motor){
+    mb_motor_read_current(motor)
```

Comment:
It's fine  to merge since we don't call the function right now, but guessing the line could cause a error (no semi-colon, recursive loop)

Reply:
Yeah I put a pass comment in for now instead. That was my mistake